<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MMT J750 400Khz I2C Pattern Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="general">
       <attribute name="title">
        <string>General Purpose</string>
       </attribute>
       <widget class="QTextEdit" name="pat_input">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>341</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_gen">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>140</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_compile">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>210</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_workbook">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>230</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="del_atp">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>290</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Del *.atp Source Vector</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="path_workbook">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_pat">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>160</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="path_pat">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="i2c_add">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>30</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>D0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D4</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="scl">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>30</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>SCL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="sda">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>70</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>SDA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern to Path:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Against IG-XL Workbook:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SCL Pin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>SDA Pinname</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>10</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>I2C Addr</string>
        </property>
       </widget>
       <widget class="QComboBox" name="numofpat">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>110</y>
          <width>31</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>90</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string># of Pat</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="atp_gen">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>270</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Auto-generate atp fiels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="pat_name">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>110</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pattern Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="offset">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>70</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Offset</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="vc5_tab">
       <attribute name="title">
        <string>VersaClock 5</string>
       </attribute>
       <widget class="QLineEdit" name="vc5_path_file">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="vc5_btn_file">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>30</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="vco_mon_off">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>VCO Monitor OFF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="vc5_btn_compile">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>170</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vc5_btn_workbook">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>190</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="del_atp_vc5">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>240</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Del *.atp Source Vector</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="vc5_path_workbook">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="vc5_btn_gen">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>60</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vc5_btn_pat">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vc5_path_pat">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Timing Commander Summary File:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern to Path:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Against IG-XL Workbook:</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="atp_gen_vc5">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>220</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Auto-generate atp fiels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="vc3_tab">
       <attribute name="title">
        <string>VersaClock 3</string>
       </attribute>
       <widget class="QPushButton" name="vc3_btn_file">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>30</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vc3_path_file">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="vc3_btn_gen">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>60</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vc3_btn_compile">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>170</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Pattern</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vc3_btn_workbook">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>190</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="del_atp_vc3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>240</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Del *.atp Source Vector</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="vc3_path_workbook">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="vc3_btn_pat">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>80</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vc3_path_pat">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>341</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Pattern to Path:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>VersaClock 3 Configuration File:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Compile Against IG-XL Workbook:</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="atp_gen_vc3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>220</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Auto-generate atp fiels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>vc5_path_file</tabstop>
  <tabstop>vc5_btn_file</tabstop>
  <tabstop>vco_mon_off</tabstop>
  <tabstop>del_atp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
